src/resonate/scheduler/scheduler.py

```diff
@@ -496,10 +502,17 @@ def _process_rfc(self, record: Record[Any], rfc: RFC) -> None:
                 if self._blocked_only_on_remote(root.id):
                     self._complete_task(root.id)
 
-    def _get_data_from_rfi(self, rfi: RFI) -> dict[str, Any]:
+    def _get_data_tags_and_headers_from_rfi(
```

Comment:
Can we move this to the bottom? We could also name something simpler such as `_get_info_from_rfi`

Reply:
I didn't have any better name. 😆 

Fixed